mantidimaging/gui/windows/recon/model.py

```diff
@@ -146,12 +148,16 @@ def run_full_recon(self, recon_params: ReconstructionParameters, progress: Progr
         # get the image height based on the current ROI
         recon = reconstructor.full(images, self.data_model.get_all_cors_from_regression(images.height), recon_params,
                                    progress)
-
-        recon = self._apply_pixel_size(recon, recon_params, progress)
+        recon = self._apply_pixel_size(recon, recon_params, progress)  # type: ignore
         return recon
 
     @staticmethod
-    def _apply_pixel_size(recon, recon_params: ReconstructionParameters, progress=None):
+    def _apply_pixel_size(recon: ImageStack | None,
+                          recon_params: ReconstructionParameters,
+                          progress: Progress | None = None) -> ImageStack | None:
+        if recon is None:
+            return None
+
```

Comment:
I think you can drop the `| None` on the `ImageStacks` here, and remove the `if recon is None:` check. THen you should not need the `ignores` above.